values.yaml

```diff
@@ -412,11 +421,52 @@ syncCatalog:
   # prepended with "consul-". (Consul -> Kubernetes sync)
   k8sPrefix: null
 
-  # k8sSourceNamespace is the Kubernetes namespace to watch for service
+  # k8sAllowNamespaces is a list of k8s namespaces to sync the k8s services from.
+  # If a k8s namespace is not included or is listed in `k8sDenyNamespaces`,
+  # pods in that k8s namespace will not be synced even if they are explicitly
+  # annotated. Use ["*"] to automatically allow all k8s namespaces.
+  #
+  # For example, ["namespace1", "namespace2"] will only allow pods in the k8s
+  # namespaces `namespace1` and `namespace2` to be synced and registered
+  # with Consul. All other k8s namespaces will be ignored.
+  # Note: `k8sDenyNamespaces` takes precedence over values defined here.
+  # Requires consul-k8s v0.12+
+  k8sAllowNamespaces: ["*"]
+
+  # k8sDenyNamespaces is a list of k8s namespaces that should not have their
+  # service synced. This list takes precedence over `k8sAllowNamespaces`.
+  # `*` is not supported because then nothing would be allowed to sync.
+  # Requires consul-k8s v0.12+.
```

Comment:
I think an example like you gave above would be great here. Like "If k8sAllowNamespaces is ["*"] and k8sDenyNamespaces is ["namespace1", "namespace2"] then all namespaces except namespace1, namespace2 will be synced.

Reply:
+1 to an example. Also, it'd be good to say what should this value be set to if you don't want to deny any namespaces. Don't know why would people want to sync services in kube-system ns, but just for completeness.

Reply:
An example is a great idea here. Updated.